### DISCUSSION TITLE
Implementation of the Negotiation Protocol in EDC Release milestone 9 vs 8

### DISCUSSION THREAD

**Discussion by JeromeCapgemini** [eclipse-edc/.github#28] [Q&A]
Hi EDC Community,
in the MERLOT Project we are studying how to integrate the EDC with our Education Marketplace Portal. We need a better understanding of what to expect from the version 8 and 9 from EDC. One of the main topic is the Contract Negotiation Protocol   https://docs.internationaldataspaces.org/dataspace-protocol/overview/readme.
What REST API are implemented and functional? (in Release milestone 9 and 8)
Where can we find those RESTful API implementation in the EDC GitHub? 
Which part of the State Machine are not working according to the IDS spec ? If it is the case , how do they behave instead ?
What is the purpose of the spi module? vs ids and the (new) dsp module ?

**Comment by mspiekermann** [comment]
With our next release we will provide an implementation of the [DSP version 2024-1](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/overview/readme), which will be the one transferred and maintained further within the Eclipse Foundation [Dataspace Working Group (EDWG)](https://dataspace.eclipse.org/).
